extensionsdk/Microsoft.Windows.DevHome.SDK/GetLocalRepositoryResult.cpp

```diff
@@ -0,0 +1,24 @@
+#include "pch.h"
+#include "GetLocalRepositoryResult.h"
+#include "GetLocalRepositoryResult.g.cpp"
+
+namespace winrt::Microsoft::Windows::DevHome::SDK::implementation
+{
+    GetLocalRepositoryResult::GetLocalRepositoryResult(winrt::Microsoft::Windows::DevHome::SDK::ILocalRepository const& repository) :
+        _repository(repository), _result(ProviderOperationStatus::Success, S_OK, hstring(), hstring())
+    {
+    }
+
+    GetLocalRepositoryResult::GetLocalRepositoryResult(winrt::hresult const& e, hstring const& displayMessage, hstring const& diagnosticText) :
```

Comment:
I believe DevHome will crash if Repository() is called after this constructor is invoked.

Reply:
`Repository()` can be called safely, but the result will contain a `nullptr`. Accessing members of the returned `ILocalRepository` without checking the return value *will* crash, as expected.